source/common/quic/envoy_quic_server_stream.h

```diff
@@ -56,6 +64,12 @@ class EnvoyQuicServerStream : public quic::QuicSpdyServerStreamBase,
   // quic::QuicSpdyServerStreamBase
   void OnConnectionClosed(quic::QuicErrorCode error, quic::ConnectionCloseSource source) override;
 
+  void clearWatermarkBuffer();
```

Comment:
?

Reply:
It was from the stream buffer PR. Merged now.